src/libcore/marker.rs

```diff
@@ -60,13 +76,16 @@ pub trait Sized {
 }
 
 /// Types that can be "unsized" to a dynamically sized type.
+///
+/// For example, the sized array type `[i8; 2]` implements `Unsize<[u8]>` and
```

Comment:
I think the RFC should still mostly reflect the current reality but I'm not super familiar with DST coercions either: https://github.com/nrc/rfcs/blob/4b99006cac67f7ad89c670bcb351d32bc0d5d33f/text/0000-dst-coercion.md